app/controllers/api/v1/lease_up_applications_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api::V1
+  # Lease Up Applications controller for access via the API
+  class LeaseUpApplicationsController < ApiController
+    def index
+      applications = lease_up_service.lease_up_listing_applications(lease_up_applications_params)
+      render json: applications
+    end
+
+    def lease_up_service
+      Force::LeaseUpService.new(current_user)
+    end
+
+    def lease_up_applications_params
+      # TODO: allow page as param
+      params.require(:listing_id)
```

Comment:
@callaghanc you might be able to help me with this. if I put

.permit(:page) in there, then I get the error ```NoMethodError - undefined method `permit' for "<<listing_id>>":String:```

Reply:
how are you adding permit ?  can you paste the code ?